draft-meynell-panrg-scion-research-questions.md

```diff
@@ -200,6 +201,14 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 * One AS belongs to many ISDs?
 
 
+## DNS Service Binding
+
+The DNS Service Binding {{RFC9460, Section 14.3}} allows to easily specify a dedicated "SCION" Service Parameter.
+
+Service Parameters allow specifying alternative IP addresses (or other parameters, such ISD/AS numbers) for a given URL.
+This would be more elegant than using DNS TXT records.
```

Comment:
Maybe you could add a bit more context about name resolution?
- 1-2 sentences explain how it was implemented today: 
```
nic@nic-mbp  % dig +short ethz.ch TXT  | grep scion
"x-sciondiscovery=8041"
"scion=64-2:0:9,129.132.230.98"
```
- Why RFC9460 is better